src/figuresClasses.ts

```diff
@@ -1,19 +1,96 @@
 export interface Figure {
-
+  color: 'red' | 'green' | 'blue',
+  shape: 'triangle' | 'circle' | 'rectangle',
```

Comment:
will be better to create enum with possible values for color and shape properties